gyucheol/2일차/문자열-내림차순으로-배치하기.js

```diff
@@ -0,0 +1,15 @@
+//1. 문자열을 배열로 변환
+//2. 배열로 변환한 요소들을 큰것부터 작은순으로 정리
+//3. 배열을 문자열로 변환한 것을 리턴함
+function solution(s) {
+    let result = s
+    .split("")
+    .sort((a,b) =>{ 
+        if(a < b) return 1; 
+        if(a > b) return -1;
+    })
+    .join('');
+    return result;
```

Comment:
```suggestion
     return s.split('').sort((a,b) =>{ 
        if(a < b) return 1; 
        else return -1;
    }).join('');
```
바로 결과가 나오는 식은 리턴으로 즉시 리턴으로 반환할 수 있습니다.
안에 조건문은 else로 사용하는 게 가독성이 더 좋습니다~ 혹여나 조건을 무조건 넣어야 한다면 else if로 작성해 주세요~